bibsonomy-uploader-cli/src/main/java/org/aksw/bibuploader/BibUpdater.java

```diff
@@ -172,5 +187,122 @@ public void diffUpdate() throws Exception {
 		}
 
 	}
+	
+	/**
+	 * Updates the account entries (B) based on the file entries (F)
+	 * @throws Exception
+	 */
+	private void updateAccount() throws Exception {
+		// load entries
+		List<Post<BibTex>> fileEntries = loadEntriesFromFile();
+		
+		// get all previously posted entries
+		List<Post<BibTex>> accountEntries = loadAllEntriesFromAccount();
+		
+		//remove duplicates from file
+		Set<String> seen = new HashSet<String>();
+		fileEntries.removeIf(entry->!seen.add(entry.getResource().getIntraHash()));
```

Comment:
This is a nice way to do that. However, maybe it would be better to be able to print warnings in case a duplicate has been found...